tests/kernel/run_ltp.pm

```diff
@@ -246,6 +246,7 @@ sub run {
     my $cmd_exclude = get_var('LTP_COMMAND_EXCLUDE') || '$^';
     my $timeout     = get_var('LTP_TIMEOUT')         || 900;
     my $is_posix    = $cmd_file =~ m/^\s*openposix\s*$/i;
+    my $is_network  = $cmd_file =~ m/^\s*net\./;
```

Comment:
@richiejp I'm still waiting for upstream reply about it, I'll test that. But if they need these variables, than it's an upstream bug (it should be added into /opt/ltp/testscripts/network.sh and have prefix 'net.').